src/corehost/cli/apphost/bundle/runner.h

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#ifndef __RUNNER_H__
+#define __RUNNER_H__
+
+#include "manifest.h"
+#include "error_codes.h"
+
+namespace bundle
+{
+    class runner_t
+    {
+    public:
+        static StatusCode extract();
+
+        static pal::string_t bundle_path;
```

Comment:
I think it would be better to:
* expose everything as methods
* be consistent - `extraction_path` is a method, `bundle_path` is a field.

I would also prefer `get_bundle_path` and so on...

Reply:
This will be subsumed by https://github.com/dotnet/core-setup/pull/6576#discussion_r289160318